securitycenter/docs/snippets_findings.py

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Examples of working with source and findings in Cloud Security Command Center."""
+
+from itertools import chain
+import os
+import pytest
+
+
+@pytest.fixture(scope="module")
+def organization_id():
+    """Get Organization ID from the environment variable """
+    return os.environ["GCLOUD_ORGANIZATION"]
+
+
+@pytest.fixture(scope="module")
+def source_name(organization_id):
+    from google.cloud import securitycenter as securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+    org_name = "organizations/{org_id}".format(org_id=organization_id)
+
+    source = client.create_source(
+        org_name,
+        {
+            "display_name": "Unit test source",
+            "description": "A new custom source that does X",
+        },
+    )
+    return source.name
+
+
+def test_create_source(organization_id):
+    """Create a new findings source. """
+    # [START create_source]
+    from google.cloud import securitycenter as securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+    # organization_id is the numeric ID of the organization. e.g.:
+    # organization_id = "111122222444"
+    org_name = "organizations/{org_id}".format(org_id=organization_id)
+
+    created = client.create_source(
+        org_name,
+        {
+            "display_name": "Customized Display Name",
+            "description": "A new custom source that does X",
+        },
+    )
+    print("Created Source: {}".format(created.name))
+    # [END create_source]
```

Comment:
I don't see any test in here (that is, any assert statements). What will happen if this fails? It won't show up as a test failure from pytest. It may (probably will) show up as an error in the test.

Reply:
This meant to be a sample, so I think failure of test is sufficient if it doesn't work?

This is consistent with in style with other samples in this repo: https://github.com/googleapis/google-cloud-python/blob/master/bigquery/docs/snippets.py